packages/calcite-components/src/components/carousel-item/carousel-item.tsx

```diff
@@ -60,8 +63,9 @@ export class CarouselItem implements LoadableComponent {
   // --------------------------------------------------------------------------
 
   render(): VNode {
+    const id = this.el.id || this.guid;
```

Comment:
I know we do this in other components but I have a question. Do we watch the `id` for changes? not sure why an id would change, but if it does then this would break right? Because we don't re-render based on the updated id changing. This may not be a realistic scenario but something we should consider. I'm sure other components would have the same issue.

Reply:
Yeah, its never come up but it could at some point.

Reply:
Looking again, this should be fine since the id is only set on host. If a user changes it on the host nothing needs to happen.